src/Microsoft.IdentityModel.Clients.ActiveDirectory/IAdalLogCallback.cs

```diff
@@ -66,7 +66,7 @@ public interface IAdalLogCallback
         /// </summary>
         /// <param name="level">Log level</param>
         /// <param name="message">message to be logged</param>
-        void Log(LogLevel level, string message);
```

Comment:
this is an API breaking change. we need to come up with other ways to support this.